modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/rel/IndexScanNode.java

```diff
@@ -84,7 +87,7 @@ public IndexScanNode(
             RowHandler.RowFactory<RowT> rowFactory,
             IgniteIndex schemaIndex,
             InternalIgniteTable schemaTable,
-            int[] parts,
+            ColocationGroup mapping,
```

Comment:
probably, it would be better to pass list of pairs [partition,term]

Reply:
Are you suggesting introducing some kind of wrapper that will hold the [int, long] pair (maybe to prevent additional boxing) or just use IgniteBiTuple<Integer, Long> and don't bother?